enterprise_access/apps/api_client/tests/test_ecommerce_client.py

```diff
@@ -0,0 +1,35 @@
+"""
+Tests for Ecommerce client.
+"""
+
+import mock
+from django.test import TestCase
+from requests import Response
+
+from enterprise_access.apps.api_client.ecommerce_client import EcommerceApiClient
+
+
+class TestEcommerceClient(TestCase):
```

Comment:
Would suggest adding some error case tests.

Reply:
so this (and the license manager) client is importing from `BaseOAuthClient` which, imports from: `from edx_rest_api_client.client import OAuthAPIClient`, because the custom logic we add is really just making the URL and then doing a GET, I personally don't think tests here are necessary (I'm making the assumption that the package we're importing from does testing of its own)

Reply:
I was thinking about the default error cases: timeout, host not found, stuff like that. If this stuff is already covered please proceed as completed.